revolv/static/mixins.scss

```diff
@@ -31,3 +31,169 @@
     -webkit-transform: $transforms;
     transform: $transforms;
 }
+
+.revolv-reveal-modal-table {
```

Comment:
I feel like this file should be for `@include`s and such and that the common modal and check styles should go in a `modal.scss`. What do you think?